depends/packages/openssl.mk

```diff
@@ -30,7 +30,7 @@ $(package)_config_opts+=no-rfc3779
 $(package)_config_opts+=no-rsax
 $(package)_config_opts+=no-sctp
 $(package)_config_opts+=no-seed
-# $(package)_config_opts+=no-sha0
+$(package)_config_opts+=no-sha0
```

Comment:
Note: this goes away when we get to OpenSSL 1.1.x

Reply:
Yeah. And we shouldn't be using anything based on SHA0 or 1...